PWGHF/D2H/TableProducer/dataCreatorCharmResoReduced.cxx

```diff
@@ -158,7 +158,7 @@ struct HfDataCreatorDV0Reduced {
   /// \param collision is the current collision
   /// \return a bitmap with mass hypotesis if passes all cuts
   template <typename V0, typename Coll, typename Tr>
-  inline uint8_t isSelectedV0(const V0& v0, const Coll& collision, const std::array<Tr, 2>& dauTracks, const std::array<int, 3>& dDaughtersIDs)
+  inline uint8_t getSelectionMapV0(const V0& v0, const Coll& collision, const std::array<Tr, 2>& dauTracks, const std::array<int, 3>& dDaughtersIDs)
```

Comment:
Add missing documentation of `dauTracks`, `dDaughtersIDs`.

Reply:
```c++
    // mass hypotesis
    if (TESTBIT(isSelected, K0s) && std::fabs(v0.mK0Short() - MassK0) > deltaMassK0s) {
      CLRBIT(isSelected, K0s);
    }
```
Why do you test the bit before checking the mass condition/

Reply:
`isSelected` is not a `bool` variable. Rename to something like `selectionMapV0`.

Reply:
Ok I will edit it